packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxNavButton/OnyxNavButton.vue

```diff
@@ -14,9 +14,9 @@ const props = withDefaults(defineProps<OnyxNavButtonProps>(), {
 
 const emit = defineEmits<{
   /**
-   * Emitted when the nav item is clicked (via click or keyboard).
+   * Emitted when the nav button is clicked (via click or keyboard).
    */
-  click: [href: string];
+  navButtonClick: [href: string];
```

Comment:
Preferably we use a emit name that describes the intent of the user.
So I would prefer to rename the emit to `navigate` (also for the navitem).
```suggestion
  navigate: [href: string];
```